### PR TITLE
use emonhub.config script from ansible

### PR DESCRIPTION
**Pull request by Basazababo**

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -31,7 +31,8 @@ sudo ln -s $GIT_PATH/conf/default/emonhub /etc/default/emonhub
 ### create folder and move settings file (unless it exists already)
 if [ ! -f /home/debian/data/emonhub.conf ] ; then
```
Comment (jlark) [inline install]: get rid of the whole if statment.

File: install [hunk 2]
```diff
@@ -29,11 +29,12 @@ sudo rm -f /etc/default/emonhub
 sudo ln -s $GIT_PATH/conf/default/emonhub /etc/default/emonhub
 
 ### create folder and move settings file (unless it exists already)
```
Comment (jlark) [inline install]: also if you're not deleting but commenting out please put in a comment saying why into the code.

**Comment by Basazababo** [comment]
Actually, there is no need to keep that. Let me delete it.

**Comment by jlark** [comment]
lgtm

**Pull request merged**